documentation/develop/onboard-upboard-offboard-users.md

```diff
@@ -205,6 +205,8 @@ browser.runtime.onInstalled.addListener(async ({ reason, temporary }) => {
 });
 {% endhighlight %}
 
+<u>Tip</u>: If your upgrade involves requesting additional permissions, see [Test permission requests](/documentation/develop/test-permission-requests) for details on how to test those changes.
```

Comment:
LGTM , thanks @rebloor ! , can you please change the `last_updated_by` at `L18` in this file and add your name  in `contributors` at `L17`.

Reply:
Thanks @championshuttler,  I have updated `last_updated_by`, I was already on the `contributors` line.